.travis.yml

```diff
@@ -66,42 +66,36 @@ before_install:
 - export CHROME_BIN=/usr/bin/google-chrome-stable
 - export DISPLAY=:99.0
 - bash -e /etc/init.d/xvfb start
-# The CHROME_SOURCE_URL is an environment variable set in the repository settings
-# of Travis. It can be found under 'Environment Variables' header here:
-# https://travis-ci.org/oppia/oppia/settings
-- wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
-- sudo dpkg -i /tmp/$(basename $CHROME_SOURCE_URL)
 
 install:
 - pushd $TRAVIS_BUILD_DIR
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
 
 script:
-- if [ $RUN_LINT == 'true' ]; then bash scripts/install_third_party.sh; python scripts/pre_commit_linter.py --path=.; fi
+- if [ "$RUN_LINT" == 'true' ]; then bash scripts/install_third_party.sh; python scripts/pre_commit_linter.py --path=.; fi
```

Comment:
No need to double quote I think?

Reply:
This gets rid of the 
`/home/travis/.travis/job_stages: line 78: [: ==: unary operator expected`
in the Travis log. I just did it cos I was doing typing out the if["$TRAVIS"... block.

Reply:
Hmm, then that's good -- let's keep it that way!